var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -56,23 +57,34 @@ class Gcc(AutotoolsPackage):
     version('4.6.4', 'b407a3d1480c11667f293bfb1f17d1a4')
     version('4.5.4', '27e459c2566b8209ab064570e1b378f7')
 
+    # Builds all default languages by default.
+    # Ada, Go, Jit, and Objective-C++ are not default languages.
+    # In that respect, the name 'all' is rather misleading.
+    variant('languages',
+            default='all',
```

Comment:
also, to be fair, i don't think there will be a lot of users installing `:4.8`. To my knowledge major linux distros all have `4.8` and above. So i would personally prefer not to stick with `all` only for `@:4.8`'s sake, instead have a workaround/filtering for older versions.